code/modules/mob/living/carbon/human/species.dm

```diff
@@ -1892,6 +1893,96 @@ GLOBAL_LIST_EMPTY(roundstart_races)
+				else if(gas_amount >= 10)
+					to_chat(humi, span_warning(pick("Your skin starts to freeze!", "You feel very cold!")))
+					var/current_stamina = humi.getStaminaLoss()
+					if(current_stamina > 25)
+						humi.adjustStaminaLoss(gas_amount * 0.15 * delta_time)
+					humi.adjust_bodytemperature(max(- gas_amount * 0.25, BODYTEMP_COOLING_MAX) * delta_time)
+					if(humi.bodytemperature < 265)
+						humi.apply_status_effect(/datum/status_effect/freon)
+			if(/datum/gas/halon)
+				if(gas_amount > 15)
+					humi.apply_status_effect(/datum/status_effect/stoned)
+			if(/datum/gas/healium)
+				if(gas_amount > 5)
+					to_chat(humi, span_notice("You feel better."))
+					humi.adjustBruteLoss(- gas_amount * 0.05 * delta_time)
+					humi.adjustFireLoss(- gas_amount * 0.015 * delta_time)
+					humi.adjustStaminaLoss(- gas_amount * 0.015 * delta_time)
+			if(/datum/gas/helium)
+				if(gas_amount > 15)
+					humi.apply_status_effect(/datum/status_effect/floaty)
+			if(/datum/gas/hypernoblium)
+				if(gas_amount > 5)
+					humi.apply_status_effect(/datum/status_effect/antimagic)
+			if(/datum/gas/nitryl)
+				if(ispodperson(humi))
+					humi.adjustToxLoss(- gas_amount * 0.5 * delta_time)
+					humi.adjustBruteLoss(- gas_amount * 0.5 * delta_time)
+					var/current_reagent = humi.reagents.get_reagent_amount(/datum/reagent/nitryl)
+					humi.reagents.add_reagent(/datum/reagent/nitryl, max(0, 1 - current_reagent))
+					continue
+				if(gas_amount > 5 && gas_amount < 10)
+					to_chat(humi, span_warning(pick("Your skin feels itchy.", "You feel like burning.")))
+				else if(gas_amount >= 10)
+					to_chat(humi, span_warning(pick("Your skin feels hot.", "You feel like burning.")))
+					humi.adjustFireLoss(gas_amount * 0.05 * delta_time)
+			if(/datum/gas/oxygen)
+				if(gas_amount > 50 && environment.temperature > 400)
+					humi.adjust_fire_stacks(gas_amount * 0.1)
+			if(/datum/gas/plasma)
+				if(isplasmaman(humi))
+					humi.adjustBruteLoss(- gas_amount * 0.5 * delta_time)
+					humi.adjustFireLoss(- gas_amount * 0.15 * delta_time)
+					humi.adjustStaminaLoss(- gas_amount * 0.15 * delta_time)
+			if(/datum/gas/pluoxium)
+				if(gas_amount > 10)
+					humi.adjustOxyLoss(- gas_amount * 0.15 * delta_time)
+			if(/datum/gas/proto_nitrate)
+				if(gas_amount > 10)
+					to_chat(humi, span_warning(pick("Your skin burns!", "You want to remove your skin!")))
+					humi.apply_status_effect(/datum/status_effect/gas_rash)
+			if(/datum/gas/tritium)
+				if(gas_amount > 1 && gas_amount < 5)
+					to_chat(humi, span_warning(pick("Your skin feels tingly.", "You feel tingly all over.")))
+				else if(gas_amount >= 5)
+					to_chat(humi, span_warning(pick("Your skin burns!", "You feel weak!")))
+					humi.radiation += gas_amount * 0.15 * delta_time
+					var/current_stamina = humi.getStaminaLoss()
+					if(current_stamina > 25)
+						humi.adjustStaminaLoss(gas_amount * 0.15 * delta_time)
+			if(/datum/gas/water_vapor)
```

Comment:
it fogs your vision

Reply:
I don't like this

Reply:
why not? i think is one of the best of them all

Reply:
water vapor is already reasonably hazardous in some scenarios, and I don't want to expand that to plasma fires alongside making the standard scenario worse

Reply:
you think i could add this behaviour to some other gas? cause i really like it, n2o maybe?

Reply:
cause i want this effect for at least one gas

Reply:
why exactly? it really incentivizes wearing hardsuits, but why?

Reply:
well not only hardsuit, you can counter it by using glasses and masks too (the ones that cover the eyes)
i like that there is one gas that fogs your vision and limit it. It's nothing majorly dangerous, just a neat quirk of a gas.
In plasmafires is not even that bad because the amount required for blindness is high and it requires a long burn before that threshold. And honestly plasmafires are not that hard to avoid/escape from, even if in the middle of them

Reply:
I don't really like effects that limit vision like this, especially in atmos senarios where seeing where you're going is important

Reply:
dont really have much interaction in atmos scenarios with water vapor, unless you like going into a delaming sm, or going into a turbine. but that can be prevented by putting holo fans at the doors of the sm.